sqlite3_opt_serialize.go

```diff
@@ -0,0 +1,88 @@
+// +build !libsqlite3 sqlite_serialize
+
+package sqlite3
+
+/*
+#ifndef USE_LIBSQLITE3
+#include <sqlite3-binding.h>
+#else
+#include <sqlite3.h>
+#endif
+#include <stdlib.h>
+#include <stdint.h>
+*/
+import "C"
+
+import (
+	"fmt"
+	"math"
+	"reflect"
+	"unsafe"
+)
+
+const (
+	SQLITEDeserializeFreeOnClose = 1
```

Comment:
Why do you need this? Can't you use the value from the sqlite3.h header file?

Reply:
I am not super familiar with cgo programming, so missed I can do that -- fixed.